### DISCUSSION TITLE
How to set options?

### DISCUSSION THREAD

**Discussion by begtostudy** [702573N/Obsidian-Tasks-Calendar#80] [Q&A]
Obsidian-Tasks-Calendar
await dv.view("tasksCalendar", {pages: "", view: "month", firstDayOfWeek: "1", ==options: "style1"==})

In Help Documents：
Hide the week number in front of each week-wrapper inside the month calendar. After deactivation, it is unfortunately no longer possible to jump directly to a desired week.

==options: "noFilename"==

I write this:
dataviewjs
await dv.view("tasksCalendar", {pages: "dv.pages().file.tasks.where(t => !t.checked && !t.tags.includes('#测试'))", view: "month", firstDayOfWeek: "1", options: 'style1,noFilename'})

Why it's wrong？

**Comment by maximecharriere** [comment]
Did you found the solution ?

**Comment by maximecharriere** [comment]
Found it, you have to write them in the same string, e.g. :
```
options: "style3 noCellNameEvent mini noFilename"
```

**Comment by gordonstrodel** [comment]
Thank you for this!
